workers/go/workflows/throughput_stress.go

```diff
@@ -0,0 +1,181 @@
+package workflows
+
+import (
+	"fmt"
+	"github.com/temporalio/omes/loadgen/througput_stress"
+	"github.com/temporalio/omes/workers/go/activities"
+	"go.temporal.io/sdk/temporal"
+	"go.temporal.io/sdk/workflow"
+	"time"
+)
+
+const (
+	UpdateSleep         = "sleep"
+	UpdateActivity      = "activity"
+	UpdateLocalActivity = "localActivity"
+	ASignal             = "someSignal"
+)
+
+// ThroughputStressWorkflow is meant to mimic the throughputstress scenario from bench-go of days
+// past, but in a less-opaque way. We do not ask why it is the way it is, it is not our place to
+// question the inscrutable ways of the old code.
+func ThroughputStressWorkflow(ctx workflow.Context, params *througput_stress.WorkflowParams) (string, error) {
```

Comment:
Bit surprised you couldn't get this with the kitchen sink one. It has concurrent nested action sets and such, but yeah it'd take additions for more capabilities.

Reply:
Honestly, it was more about making it obvious than anything else. KitchenSink is kinda like bench go and as discussed I'm not sure I really like that much abstraction.

I think it's useful for fuzz testing or something, for sure, but that's not quite the purpose of this one.

Reply:
Works for me, but I do think throughputstress stuff needs to be together and not separated

Reply:
Done